### PR TITLE
Updated Okta & Spring boot starter versions to latest

### PR DESCRIPTION
**Pull request by arvindkrishnakumar-okta**
Updated the below dependency versions:

- spring-boot-starter-parent from 2.1.5.RELEASE -> 2.2.5.RELEASE
- okta-spring-boot-starter from 1.2.1 -> 1.4.0

### CODE REVIEW

File: custom-login/README.md [hunk 1]
```diff
@@ -26,7 +26,8 @@ Plug these values into the `mvn` commands used to start the application.
 cd custom-login
 mvn -Dokta.oauth2.issuer=https://{yourOktaDomain}/oauth2/default \
     -Dokta.oauth2.clientId={clientId} \
-    -Dokta.oauth2.clientSecret={clientSecret}
+    -Dokta.oauth2.clientSecret={clientSecret} \
+    -Dokta.oauth2.postLogoutRedirectUri={postLogoutRedirectUri}
```
Comment (bdemers) [inline custom-login/README.md]: We probably don't want to add this one here, but if we do add it, we would need to include details on how to configure it.
My suggestion is to move this out to a different pr (if it needs to be added)

File: custom-login/pom.xml [hunk 2]
```diff
@@ -41,7 +41,7 @@
         <dependency>
             <groupId>com.okta.spring</groupId>
             <artifactId>okta-spring-boot-starter</artifactId>
-            <version>1.2.1</version>
+            <version>1.3.0</version>
```
Comment (bdemers) [inline custom-login/pom.xml]: 1.4.0 is the latest version

File: custom-login/README.md (hunk 1 above)
Comment (arvindkrishnakumar-okta) [reply custom-login/README.md]: Addressed this in next commit.

File: custom-login/pom.xml (hunk 2 above)
Comment (arvindkrishnakumar-okta) [reply custom-login/pom.xml]: Addressed.

**Comment by bdemers** [comment]
You might need to bump the tck version too. Are the ITs passing locally for you?

**Comment by arvindkrishnakumar-okta** [comment]
> You might need to bump the tck version too. Are the ITs passing locally for you?

Good point.. it fails in local but with a different error.

**Comment by arvindkrishnakumar-okta** [comment]
> I a couple of minor issues, see above

@bdemers Addressed them all, please review.
